source/jormungandr/jormungandr/autocomplete/kraken.py

```diff
@@ -77,6 +83,13 @@ def get(self, request, instance):
         if len(resp.places) == 0 and request['search_type'] == 0:
             req.places.search_type = 1
             resp = instance.send_and_receive(req)
+
+        # add bano into the feed publisher if not existent
+        has_bano = next((True for f in resp.feed_publishers if f.id == FEED_PUBLISHER_BANO['id']), None)
```

Comment:
for bragi, i did it [here](https://github.com/CanalTP/navitia/pull/2184/files#diff-204c59b4be2984c009079df1061d66f7R279).